app/controllers/api/entities/label.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module API
+  module Entities
+    class Label < Grape::Entity
+      format_with(:iso_timestamp, &:iso8601)
+
+      expose :key, documentation: { type: 'String', desc: 'Label key. [a-z0-9_-]+ should be used. Min - 3, max - 255 characters.' }
+      expose :value, documentation: { type: 'String', desc: 'Label value. Min - 3, max - 255 characters.' }
+      expose :scope, documentation: { type: 'String', desc: "Label scope: 'general' or 'restricted'" }
+      expose :account_id,
```

Comment:
could he avoid returning account_id in this entity?

Reply:
sure

Reply:
done